README.md

```diff
@@ -8,7 +8,80 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running the demo on Windows
 
+Prerequisites:
+- [Docker desktop](https://docs.docker.com/docker-for-windows/install/)
+
+### Step 1
+First test that the docker installation is working. Open a Powershell prompt (press Windows+S and type Powershell) and run:
+```shell
+docker run hello-world
+```
+This should show a message that your Docker installation is working correctly. If so, we can proceed to the installation of WorkingTitleCloseReader, otherwise we suggest to check the steps in the [Docker installation guide](https://docs.docker.com/docker-for-windows/install/).
+
+### Step 2
+[Download](https://github.com/ADAH-EviDENce/WorkingTitleCloseReader/archive/master.zip) a copy of WorkingTitleCloseReader archive and extract its contents on your machine.
+
+### Step 3
+Open a Powershell prompt:
+Change your current working directory to where you extracted the files. For instance:
+
+```shell
+cd C:\Users\JohnDoe\Downloads\WorkingTitleCloseReader-master\WorkingTitleCloseReader-master
+```
+
+The demo can be started with the below command. Keep this Powershell window open and running during the demo.
+
+```shell
+docker-compose --env-file getuigenverhalen.env up --build
```

Comment:
Use env var instead of file
```suggestion
$Env:EXPERIMENT="getuigenverhalen"
docker-compose up --build
```

Reply:
I installed Docker Desktop with wsl 2 integration enabled and the searches on the frontend are failing,. Seems to be same errors as in #77